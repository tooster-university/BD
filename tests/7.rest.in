{"open": {"database": "student", "login": "app", "password": "qwerty"}}
{"downvote": {"timestamp": 49997356, "member": 10, "password": "DVD9OSZB", "action": 190}}
{"projects": {"timestamp": 82710095, "member": 10, "password": "DVD9OSZB"}}
{"support": {"timestamp": 116867563, "member": 5, "password": "7RFQR", "action": 148, "project": 355, "authority": 430}}
{"actions": {"timestamp": 135683530, "member": 10, "password": "L1R4YXHV"}}
{"votes": {"timestamp": 136705954, "member": 6, "password": "L0H5S3Z1"}}
{"trolls": {"timestamp": 182420594}}
{"support": {"timestamp": 234482784, "member": 10, "password": "DVD9OSZB", "action": 147, "project": 373, "authority": 399}}
{"downvote": {"timestamp": 272355938, "member": 6, "password": "L0H5S3Z1", "action": 188}}
{"projects": {"timestamp": 333095595, "member": 5, "password": "5QRKABP0L"}}
{"protest": {"timestamp": 360384686, "member": 5, "password": "7RFQR", "action": 143, "project": 347, "authority": 428}}
{"projects": {"timestamp": 403651048, "member": 10, "password": "SYAO18P5IH"}}
{"upvote": {"timestamp": 436242481, "member": 10, "password": "DVD9OSZB", "action": 194}}
{"actions": {"timestamp": 463562306, "member": 5, "password": "5QRKABP0L"}}
{"upvote": {"timestamp": 507892010, "member": 10, "password": "L1R4YXHV", "action": 111}}
{"support": {"timestamp": 558113623, "member": 5, "password": "5QRKABP0L", "action": 149, "project": 356, "authority": 386}}
{"support": {"timestamp": 587312666, "member": 5, "password": "5QRKABP0L", "action": 194, "project": 354, "authority": 452}}
{"support": {"timestamp": 624697679, "member": 5, "password": "7RFQR", "action": 159, "project": 300, "authority": 371}}
{"protest": {"timestamp": 627263587, "member": 6, "password": "L0H5S3Z1", "action": 121, "project": 286, "authority": 439}}
