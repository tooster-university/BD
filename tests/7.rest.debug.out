{"status": "OK"}
{"status": "ERROR", "debug": "User can't cast vote on action that does not exist!"}
{"status": "ERROR", "debug": "ERROR:  Member must be leader but doesnt exist"}
{"status": "ERROR", "debug": "ERROR:  Incorrect member password"}
{"status": "ERROR", "debug": "ERROR:  Member must be leader but doesnt exist"}
{"status": "ERROR", "debug": "ERROR:  Member must be leader but doesnt exist"}
{"status": "OK", "data": []}
{"status": "OK"}
{"status": "ERROR", "debug": "User can't cast vote on action that does not exist!"}
{"status": "ERROR", "debug": "ERROR:  Incorrect member password"}
{"status": "ERROR", "debug": "ERROR:  Incorrect member password"}
{"status": "ERROR", "debug": "ERROR:  Incorrect member password"}
{"status": "ERROR", "debug": "ERROR:  Member inactive"}
{"status": "ERROR", "debug": "ERROR:  Incorrect member password"}
{"status": "ERROR", "debug": "ERROR:  Incorrect member password"}
{"status": "ERROR", "debug": "ERROR:  Incorrect member password"}
{"status": "ERROR", "debug": "ERROR:  Incorrect member password"}
{"status": "ERROR", "debug": "ERROR:  Incorrect member password"}
{"status": "OK"}
